util/cron/test-linux64-python35.bash

```diff
@@ -10,4 +10,13 @@ export CHPL_NIGHTLY_TEST_CONFIG_NAME="linux64-python35"
 
 set_python_version "3.5"
 
+# Check correct Python version loaded
+expected_python_version="3.5.2"
+actual_python_version=$(python3 --version | cut -d' ' -f2)
+if [ "$actual_python_version" != "$expected_python_version" ]; then
+  echo "Wrong Python version"
+  echo "Expected Version: $expected_python_version. Actual Version: $actual_python_version"
+  exit 2
+fi
```

Comment:
I'd make lines 15-20 into a helper function in `common-python.bash` and call that function with the expected version number, instead of duplicating it in each job.

Other than that, this looks good to me!

Reply:
I went ahead and put the check into the existing `setup_python_version` function and made it take a full version string, so we don't have to specify the Python version twice